src/core_plugins/getting_started/public/components/getting_started/getting_started.html

```diff
@@ -0,0 +1,135 @@
+          <div class="kuiCard__descriptionTitle">
+            <h2 class="kuiSubTitle">
+              First, add your data
+            </h2>            
+          </div>
+
+          <div class="kuiCard__descriptionText gettingStartedCard__descriptionText">
+            <div class="gettingStartedLogoRow">
+              <img src="{{ gettingStarted.imageUrls.beatsLogo }}" />
+              <img src="{{ gettingStarted.imageUrls.logstashLogo }}" />
+            </div>
+          </div>
+        </div>
+
+        <div class="kuiCard__footer">
+          <p class="kuiText">Pull in data from the <a href="">Elasticsearch API</a>, <a href="">Beats</a>, <a href="">Logstash</a>.
+          Not sure how to do that? <a href="">Read this introduction.</a></p>
+        </div>
+      </div>
+
+      <div class="kuiCard kuiCardGroup__card gettingStartedCard">
+        <div class="kuiCard__description">
+          <div class="kuiCard__descriptionTitle">
+            <h2 class="kuiSubTitle">
+              Visualize and explore
+            </h2>
+          </div>
+
+          <div class="kuiCard__descriptionText gettingStartedCard__descriptionText">
+            <div class="gettingStartedLogoRow">
+              <img src="{{ gettingStarted.imageUrls.dashboardIcon }}" />
+            </div>
+          </div>
+        </div>
+
+        <div class="kuiCard__footer">
+          <p class="kuiText">Once you've added data, you'll need to
+          <a
+            href="#/management/kibana/index"
+            ng-click="gettingStarted.recordOptOut()"
+          >
+            configure an index pattern
+          </a>
+          for your data before using our Discover and Visualize applications.</p>
+        </div>
+      </div>
+
+      <div
+        class="kuiCard kuiCardGroup__card gettingStartedCard"
+        ng-show="gettingStarted.hasManageAndMonitorMessages"
+      >
+        <div class="kuiCard__description">
+          <div class="kuiCard__descriptionTitle">
+            <h2 class="kuiSubTitle">
+              Manage and monitor
+            </h2>
+          </div>
+
+          <div class="kuiCard__descriptionText gettingStartedCard__descriptionText">
+            <div class="gettingStartedLogoRow">
+              <img src="{{ gettingStarted.imageUrls.shieldIcon }}" />
+            </div>
+          </div>
+        </div>
+
+        <div class="kuiCard__footer">
+          <p
+            class="kuiText"
+            ng-if="gettingStarted.manageAndMonitorMessages"
+            ng-bind-html="gettingStarted.manageAndMonitorMessages | trustAsHtml"></p>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="kuiViewContentItem kuiVerticalRhythm">
+    <div class="kuiFieldGroup">
+      <div class="kuiFieldGroupSection">
+        <p class="kuiText">Just want to see what Kibana is capable of doing?</p>
+      </div>
+
+      <div class="kuiFieldGroupSection">
+        <button class="kuiButton kuiButton--primary">
```

Comment:
Sounds like a great idea, the kuiButton classes are designed to support that use case.